packages/scipy/meta.yaml

```diff
@@ -50,8 +52,14 @@ build:
     -L$(NUMPY_LIB)/core/lib/
     -L$(NUMPY_LIB)/random/lib/
     -fexceptions
+
+  # Exclude tests via Meson's install tags functionality.
+  unvendor-tests: true
+  # install-args=--tags=runtime,python-runtime,devel
+  # Disable when running tests, enable when a PR is ready, i.e., building for distribution.
```

Comment:
I haven't been able to get the[ install tags functionality](https://docs.scipy.org/doc/scipy/building/redistributable_binaries.html) working properly. Would you have any suggestions here, @rgommers? I would like that someone updating the package recipe can have a means of disabling the functionality momentarily to be able to trigger the tests, i.e., both the ones running on CircleCI and the ones running on GHA through the `[scipy]` commit marker established in #4935, and once the recipe is ready and passes all tests, enable it back so that the test-tagged modules can be not built when merging the recipe and we get a smaller wheel (and I can document that process better in a comment above this field).

However, it doesn't seem to work, and it still tries to build the modules that have been removed in patch `0017` (and fails in doing so). Could this be a bug in meson-python or am I using this functionality incorrectly? I can push another commit if needed with the install tags enabled to provide a failing job.

Reply:
Maybe it's related to https://github.com/scipy/scipy/pull/21225, but I don't see why it wouldn't respect `install_tag: tests` either way.

Reply:
It's not a bug - all modules get built. Install tags are purely an install-time feature, selecting what targets get installed during the `meson install` step (which is separate from `meson compile`). If you don't want a target to be built at all, you have to patch it out still.